### PR TITLE
chore: replace twitter urls with x (unrealapex)

### LINKED ISSUE #5540: Update URLs and logos with X design (former Twitter)
Update URLs and logos with X design (former Twitter). Now that we can use fontawesome-6, this change should be easy.
I can create a fork and work on this if you guys agree.

https://fontawesome.com/icons/x-twitter?f=brands&s=solid

_Originally posted by @aescanes in https://github.com/monkeytypegame/monkeytype/discussions/5478_

### PR DESCRIPTION
**Pull request by unrealapex**
Change Monkeytype Twitter links to X.

Closes #5540

### CODE REVIEW

File: frontend/src/html/pages/about.html [hunk 1]
```diff
@@ -200,12 +200,12 @@ <h2>
...
         class="button"
-        href="https://twitter.com/monkeytypegame"
+        href="https://x.com/monkeytypegame"
         target="_blank"
         rel="noreferrer noopener"
       >
-        <div class="fab fa-twitter"></div>
-        twitter
+        <div class="fab fa-x-twitter"></div>
+        x
```
Comment (aescanes) [inline frontend/src/html/pages/about.html:208]: Maybe this should be "X" instead of "x"

File: frontend/src/ts/elements/profile.ts [hunk 2]
```diff
@@ -267,11 +267,11 @@ export async function update(
       const twitter = profile.details?.socialProfiles.twitter ?? "";
       if (twitter) {
         socialsEl.append(
-          `<a href='https://twitter.com/${Misc.escapeHTML(
-            twitter
+          `<a href='https://x.com/${Misc.escapeHTML(
+            x
```
Comment (aescanes) [inline frontend/src/ts/elements/profile.ts:271]: Maybe this should be "X" instead of "x"

**Comment by fehmer** [comment]
Hi @unrealapex , we are still using fontawesome 5.

**Comment by unrealapex** [comment]
Thank you for letting me know. Should we bump fontawesome's version or keep this on hold until a later date?

**Comment by unrealapex** [comment]
Sounds good. Will do.

**Review by Miodec — APPROVED** [review]
Thanks

**Pull request merged**